environment

```diff
@@ -52,42 +50,54 @@ EXPORT_ENV_VARS_TO_LAMBDA_ARRAY=(
 set -a
 EXPORT_ENV_VARS_TO_LAMBDA=${EXPORT_ENV_VARS_TO_LAMBDA_ARRAY[*]}
 DSS_DEPLOYMENT_STAGE=dev
+STAGE="${DSS_DEPLOYMENT_STAGE}"
 ADMIN_USER_EMAILS_SECRETS_NAME=admin_user_emails
 AWS_DEFAULT_OUTPUT=json
-AWS_DEFAULT_REGION=us-east-1
+AWS_DEFAULT_REGION=us-west-2
```

Comment:
can we migrate everything back to us-east-1?

Reply:
Will do. The [gi wiki](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account) mentions that most stuff exists in `us-west-2` so that was the reason I switched them out.

Reply:
Fixed with 46a0a5b